news/20210315163230.misc

```diff
@@ -0,0 +1 @@
+Implement reading from details.txt to display interface version.
```

Comment:
In the commit message for "devices: Collect data from details.txt", I see "MSD" again. I'd recommend spelling it out.